packages/graphback-core/src/annotations/parser.ts

```diff
@@ -2,9 +2,9 @@ import { FieldDefinitionNode, GraphQLField, GraphQLObjectType, TypeDefinitionNod
 import { parseAnnotations } from "graphql-metadata";
 
 export const parseDbAnnotations = (node: TypeDefinitionNode | FieldDefinitionNode | GraphQLObjectType | GraphQLField<any, any>): any => {
-    return node.description ? parseAnnotations('db', String(node.description)) : undefined;
+    return node.description ? parseAnnotations('db', String(node.description)) : {};
```

Comment:
[Question] Motivation for this change?

Reply:
Saves having to check if the returned `annotations` is defined after returning.

`if (annotations && annotations.name)` => `if (annotations.name)`.

The only place we really cared about having an undefined annotation is with the `@model` marker.